modmesh/app/bad_euler1d.py

```diff
@@ -37,10 +37,8 @@
 
 
 def load_app():
-    view.app().pytext.code = """
-# Need to hold the win object to keep PySide alive.
-win, svr = mm.app.bad_euler1d.run(animate=True, interval=10)
-""".lstrip()
+    cmd = "win, svr = mm.app.bad_euler1d.run(animate=True, interval=10)"
```

Comment:
`pycon` takes a one liner.  It does not yet have a check in the engine so that we can fine-tune the implementation.